xds/src/main/java/io/grpc/xds/internal/sds/ServerWrapperForXds.java

```diff
@@ -109,16 +250,54 @@ private void removeServerWatcher() {
     }
   }
 
+  // if the IOException indicates we can rebuild delegate and retry start...
+  private static boolean isRetriableErrorInDelegateStart(IOException ioe) {
+    Throwable cause = ioe.getCause();
```

Comment:
Is it possible ioe itself is BindException?

Reply:
Hmmm. Looking at the current structure the IOException wraps a SocketException which is a BindException and I think that's what happens for most socket related IO errors. But good point and it doesn't hurt to also check for the IOException. Will do